src/pages/HomeParticipant.vue

```diff
@@ -121,17 +123,18 @@ const consentType = ref('');
 const consentParams = ref({});
 
 const isLevante = import.meta.env.MODE === 'LEVANTE';
+
+const { mutateAsync: updateConsentStatus } = useUpdateConsentMutation();
```

Comment:
This is interesting

Reply:
It really is – very important here that we use `mutateAsync` and not the usual `mutate` as we need it to throw any errors it encounters in order to catch them in the `<ConsentModal />`